conf/machine/include/qemuriscv.inc

```diff
@@ -14,7 +14,7 @@ GDBVERSION = "riscv"
 
 SERIAL_CONSOLES = "115200;ttyS0"
 
-TCLIBC = "glibc"
+TCLIBC ?= "glibc"
```

Comment:
we do not support musl yet, thats the reason to hardcode

Reply:
Same point as the freedome-u540 change

Reply:
Remove it completely.